ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -96,8 +119,8 @@
         chdir: ~/test-suite
         creates: ~/test-suite/reports
       environment:
-        LOGGING_ENDPOINT: https://staging-log-api.newrelic.com/log/v1
-        NERD_GRAPH_URL: https://staging-api.newrelic.com/graphql
+        LOGGING_ENDPOINT: https://log-api.newrelic.com/log/v1
+        NERD_GRAPH_URL: https://api.newrelic.com/graphql
```

Comment:
You sure about this?

Reply:
what do you mean? staging is protected now, we can't reach them from ec2 instances